core/jazz_admin/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "admin",
+  "version": "0.0.1",
+  "description": "Admin -api to retrive installer-vars.json for admin user",
```

Comment:
Update description to "Core API to perform administrative tasks"

Reply:
done